doc/nrf/known_issues.rst

```diff
@@ -1022,6 +1066,23 @@ DRGN-15382: The SoftDevice Controller cannot be qualified on nRF52832
 
   **Workaround:** Upgrade to v1.5.1 or use the master branch.
 
+.. rst-class:: v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-13029: The application will not immediately restart a connectable advertiser after a high duty cycle advertiser times out.
+  In some cases the host may restart advertising sooner than the SoftDevice Controller is able to free its connection context.
+
+  **Workaround** Wait some time before restarting the advertiser.
+
+.. rst-class:: v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-14008: The advertising data is cleared every time the advertising set is configured.
+  This causes the "HCI LE Set Extended Advertising Parameters" command to accept data which cannot be fit within the advertising interval instead of returning "Packet Too Long (0x45)".
```

Comment:
```suggestion
DRGN-14008: The advertising data is cleared every time the advertising set is configured.
  This causes the "HCI LE Set Extended Advertising Parameters" command to accept data which cannot be fit within the advertising interval instead of returning "Packet Too Long (0x45)".
  This would only occur if the advertising set is configured to use maximum data length on LE Coded PHY with an advertising interval less than 30 milliseconds.
```